src/prefect/runtime/variables.py

```diff
@@ -0,0 +1,33 @@
+import sys
```

Comment:
Does `prefect.runtime` make sense for variables? Can't they be retrieved outside of flow runs? It seems like `prefect.variables` makes more sense?